Document Synthetics multispace CCS settings (#7571)
elastic/docs-content: solutions/observability/synthetics/configure-settings.md

Reviewer: This section is overall quite hard to read/understand and I think fails at explaining the core things clearly. It's not in the scope of your PR here, but I think it's worth creating an issue to rewrite at least the Alerting section, and/or see if it needs its own page.
If you update a rule, does the change apply to all monitors it's active on, or only new ones?
We say we can disable them for individual monitors but don't mention the option that controls it, etc.

Author: Opened: https://github.com/elastic/docs-content-internal/issues/1527

Reviewer 2: > If you update a rule, does the change apply to all monitors it's active on, or only new ones?

All monitors, existing and new.

Alerting enables you to detect complex conditions using **rules** across Observability and send a notification using **connectors**.

> When you create a new synthetic monitor, Synthetics applies new default rules to the monitor. To edit the default rules:

Reviewer: ```suggestion
When you create a new synthetic monitor, Synthetics applies default rules to the monitor. To edit the default rules:
```

> 1. Click **Alerts** in the global header.

Reviewer: ```suggestion
1. Click **Alerts** in the application menu.
```
The global header is this 

<img width="1659" height="44" alt="Image" src="https://github.com/user-attachments/assets/6359fa05-22ad-49f6-bd06-f20ed1d2ff9f" />

See https://www.elastic.co/docs/contribute-docs/style-guide/ui-writing#naming-kibana-ui-elements

Author: Oh ok, thanks!

2. Select a rule to open a panel where you can edit the rule’s configuration:

    * **Monitor status rule** for receiving notifications for errors and outages
    * **TLS certificate rule** for receiving notifications when one or more of your HTTP or TCP lightweight monitors has a TLS certificate expiring within a specified threshold or when it exceeds an age limit

However, the automatically created Synthetics internal alert is intentionally preconfigured, and some configuration options can’t be changed. For example, you can’t change how often it checks the rule.

If you need specific alerting behavior, set up a different rule. To view all existing rules or create a new rule:

> 1. Click **Alerts** in the global header.

Reviewer: ```suggestion
1. Click **Alerts** in the application menu.
```

:::

- The **Elastic managed locations enabled** toggle controls whether your Synthetics monitors are permitted to run from Elastic's globally distributed, cloud-hosted testing infrastructure. If enabled, the key can push monitors to both Elastic-managed and private locations. If disabled, the key is restricted to private locations only.
- The **Spaces** menu allows you to select the space where your API keys are available.

## Remote clusters [synthetics-settings-remote-clusters]
```{applies_to}

> stack: ga 9.5+

Reviewer: ```suggestion
stack: ga 9.5+
serverless: unavailable
```

```

In the **Remote clusters** tab, you can configure {{ccs}} ({{ccs-init}}) settings so that Synthetics can include monitor data from remote {{es}} clusters alongside your local monitors.

> These settings apply to one or more {{kib}} spaces. Changes you make here affect all spaces that have access to these settings.

Reviewer: I think this sentence relates to the last subsection here but it's not super clear to me what it means. Is it: "You can apply these settings to other {{kib}} spaces." or "These settings only apply to the default space by default, but you can specify other spaces to share that configuration across them"? or something else.

Author: There is one CCS settings document per deployment. The combo box where you choose spaces controls which Kibana spaces can see that one shared configuration (and, therefore, the config applies to them). I rephrased it for clarity.

Reviewer 2: Regarding your rephrased version looks mostly correct, but I left some feedback about `deployment-wide` wording. I understand how you mean it, but this could read as "applies to the whole deployment", which is not true by default. The settings only apply in the spaces you grant access to, a space without access sees the defaults (CCS off). I left a comment with my [suggestion](https://github.com/elastic/docs-content/pull/7571/changes#r3675455318).

Author: Thanks, applied

### Source settings [synthetics-settings-remote-clusters-source]

Use the **Use all remote clusters** toggle to turn on or turn off {{ccs}} for Synthetics. When enabled, monitors from remote clusters appear alongside local monitors in the Synthetics UI.

### Remote clusters [synthetics-settings-remote-clusters-select]

When **Use all remote clusters** is disabled, use the **Select remote clusters** combo box to select which remote clusters Synthetics queries. Each cluster is shown with a connected or disconnected status indicator.

> To configure remote clusters, go to **{{stack-manage-app}} → Remote Clusters**.

Reviewer: These 2 seem to be mutually exclusive and do the same thing: they control which remove clusters to fetch monitors from: turn on "Use all remote clusters", or pick individual clusters from the dropdown. Because the first option disables the second one. So we can simplify here.

Author: I rewrote it and simplified the "Remote clusters" section

Reviewer 2: I left a small comment [here](https://github.com/elastic/docs-content/pull/7571/changes#r3675296864) (instead of `and` I suggest we use `or`)

### Spaces with access [synthetics-settings-remote-clusters-spaces]

> Use the **Spaces** combo box to control which {{kib}} spaces these {{ccs-init}} settings apply to. You can select individual spaces, multiple spaces, or **All spaces** to share the settings across your entire deployment. Removing a space from the selection hides these settings in that space.

Reviewer: "individual spaces" (plural) / "multiple spaces" means the same thing. From what I see you either pick spaces individually, or select All spaces.

I'm not clear about the "removing..." part. Are the settings actually hidden (for me hidden means you can't configure it there anymore)? Or does that space just get to keep its own individual ccs configuration, not shared?

Author: There are three intended modes: single space, multiple spaces, or all spaces. But you're right in that the UI implements all three through one combo box (you just pick how many spaces). I changed the phrasing to a more accurate one.

> If you leave the field empty, the current sharing configuration remains unchanged.

Reviewer: So we shouldn't even have the current space listed?

Author: It means if you select nothing, whatever was selected before is still seeing the shared config. It **does not mean** that the config is now not being used in any space. This is counter-intuitive, so I made it a note and rephrased it for clarity.

Reviewer 2: I agree that the copy `Leave empty to keep the current sharing configuration.` was the root of the confusion. The field is a pre-filled multi-select, yet an empty selection is silently treated as "no change" (to keep the settings object in at least one space). You'd expect that emptying the list and saving removes all spaces. Instead, an empty list means "don't change anything" — which is surprising, so the help text exists to explain it.

The docs note now documents this accurately, so no change is needed here —I've filed a follow-up to fix it at the source — [elastic/kibana#281577](https://github.com/elastic/kibana/issues/281577): require at least one space (validate/disable Save when empty) instead of treating empty as a no-op. Once it ships, the docs can drop the empty-save explanation so UI and docs stay aligned.

Author: Thank you, I'll keep an eye out for that

These settings are stored as a single shared configuration — one per deployment, not a separate copy per space. Use the **Spaces** field to control which {{kib}} spaces it applies to.

::::{note}
To edit these settings, you must have the **All** privilege for the **Synthetics and Uptime** feature in **{{stack-manage-app}} → Roles → {{kib}} privileges → {{observability}}**. Users with only the **Read** privilege see the form as read-only.
::::

### Source settings [synthetics-settings-remote-clusters-source]

Use the **Use all remote clusters** toggle or **Select remote clusters** combo box to control which remote clusters Synthetics queries:

- Turn on **Use all remote clusters** to include monitor data from all configured remote clusters.
- Turn off **Use all remote clusters** and use the **Select remote clusters** combo box to choose specific clusters. Each cluster is shown with a connected or disconnected status indicator.

To configure remote clusters, go to **{{stack-manage-app}} → Remote Clusters**.

### Spaces with access [synthetics-settings-remote-clusters-spaces]

> Use the **Spaces** combo box to control which {{kib}} spaces can access these {{ccs-init}} settings. Select one or more specific spaces, or select **All spaces** to make the settings available across your entire deployment. Removing a space makes the settings inaccessible there, meaning that users in that space see default {{ccs-init}} settings instead.

Reviewer 2: `Removing a space makes the settings inaccessible there, meaning that users in that space see default {{ccs-init}} settings instead.`

I know we mentioned that CCS is off by default. Just wondering if it makes sense to   make it here more explicit and say that instead:

`Removing a space makes the settings inaccessible there, meaning that users in that space don't see remote clusters (or in that space CCS is disabled).`

:::

- The **Elastic managed locations enabled** toggle controls whether your Synthetics monitors are permitted to run from Elastic's globally distributed, cloud-hosted testing infrastructure. If enabled, the key can push monitors to both Elastic-managed and private locations. If disabled, the key is restricted to private locations only.
- The **Spaces** menu allows you to select the space where your API keys are available.

## Remote clusters [synthetics-settings-remote-clusters]
```{applies_to}
stack: ga 9.5+
serverless: unavailable
```

In the **Remote clusters** tab, you can configure {{ccs}} ({{ccs-init}}) settings so that Synthetics can include monitor data from remote {{es}} clusters alongside local monitors.

> These are deployment-wide settings stored as a single shared configuration. You control which {{kib}} spaces have access to them using the **Spaces** field.

Reviewer 2: ```suggestion
These settings are stored as a single shared configuration — one per deployment, not a separate copy per space. Use the Spaces field to control which {{kib}} spaces it applies to.
```

### Source settings [synthetics-settings-remote-clusters-source]

> Use the **Use all remote clusters** toggle and **Select remote clusters** combo box to control which remote clusters Synthetics queries:

Reviewer 2: Instead of `and` I suggest we use `or`:

Use the "Use all remote clusters" toggle or "Select remote clusters" combo box

- Turn on **Use all remote clusters** to include monitor data from all configured remote clusters.
- Turn off **Use all remote clusters** and use the **Select remote clusters** combo box to choose specific clusters. Each cluster is shown with a connected or disconnected status indicator.

To configure remote clusters, go to **{{stack-manage-app}} → Remote Clusters**.

### Spaces with access [synthetics-settings-remote-clusters-spaces]

Use the **Spaces** combo box to control which {{kib}} spaces can access these {{ccs-init}} settings. Select one or more specific spaces, or select **All spaces** to make the settings available across your entire deployment. Removing a space makes the settings inaccessible there, meaning that users in that space see default {{ccs-init}} settings instead.

::::{note}

> The **Spaces** field shows the current sharing configuration when you open the tab. Clearing the field and saving leaves that configuration unchanged, meaning it does not remove access from existing spaces. To update which spaces have access, select the new spaces explicitly.

Reviewer 2: ```suggestion
The **Spaces** field shows the current sharing configuration when you open the tab — your current space by default. Clearing the field and saving leaves that configuration unchanged, meaning it does not remove access from existing spaces. To update which spaces have access, select the new spaces explicitly.
```

::::

::::{note}

> <!-- TODO: Confirm which role/privilege grants access to edit these settings. The capability required is `uptime.configureSettings` — clarify which built-in role includes this. -->

Author: @mgiota Could you help with this? Is it available for every user? I'm not sure

Reviewer 2: Editing these settings requires the **All privilege** for the **Synthetics and Uptime** feature (Stack Management -> Roles -> Kibana privileges -> Observability). It's the same privilege that controls editing all other Synthetics settings. Users with only **Read** see the form as read-only. It's not available to every user by default.

Author: Thanks, useful info, I added it as a note

To edit these settings, you must have the required Synthetics configuration privileges. If the form is read-only, contact your administrator.
::::

> <!-- TODO: Confirm the default space scope when these settings are first saved on a new deployment. -->

Author: @mgiota What are the default settings? Maybe it's worth mentioning.

Reviewer 2: Cross-cluster search is **off by default**. So until you configure it, only local monitors appear.

**Space scope on first save**: the settings are anchored to the **space you're currently** in when you first save. You can use the Spaces field to extend access to other spaces.

##### Default space
<img width="750" height="328" alt="Image" src="https://github.com/user-attachments/assets/b5c6bf22-dedc-444e-80b4-36807b290e15" />

#### Space b
<img width="750" height="330" alt="Image" src="https://github.com/user-attachments/assets/4062f4f9-dfce-4523-909e-0c248e441bd3" />

Author: Great, I mentioned it on line 125
